SDK/AppCenter/Microsoft.AppCenter.Android/HttpResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.AppCenter
+{
+    using AndroidHttpResponse = Com.Microsoft.Appcenter.Http.AndroidHttpResponse;
```

Comment:
Move the using outside of the namespace.

Reply:
This is a pattern that we have in another place. I assume it isn't consistent.

Reply:
Fixed it though since this is new code.